src/js/jsx/sections/layers/LayersGroup.jsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2016 Adobe Systems Incorporated. All rights reserved.
+ *  
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"), 
+ * to deal in the Software without restriction, including without limitation 
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, 
+ * and/or sell copies of the Software, and to permit persons to whom the 
+ * Software is furnished to do so, subject to the following conditions:
+ *  
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *  
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+ * DEALINGS IN THE SOFTWARE.
+ * 
+ */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var React = require("react"),
+        Fluxxor = require("fluxxor"),
+        FluxMixin = Fluxxor.FluxMixin(React),
+        classnames = require("classnames");
+
+    var DummyLayerFace = require("./DummyLayerFace"),
+        LayerFace = require("./LayerFace");
+
+    var LayersGroup = React.createClass({
+        mixins: [FluxMixin],
+
+        getDefaultProps: function () {
+            return {
+                depth: 0,
+                isRoot: false
+            };
+        },
+
+        render: function () {
```

Comment:
Good point. I included the solution in my previous PR (https://github.com/adobe-photoshop/spaces-design/commit/05c90a51684af68897a62a2efa4e9f017fed820f#diff-599bf64b66bfd695d31114df0d8d0c63R172), which is a layer diff algorithm in the `LayerPanel` or in the root `LayerGroup`, as we want to avoid repeating the diff in each nested `LayerGroup` when the deepest leaf changed. However, it adds extra complexity, so we decided to not do it.